crypto/wolfssl/Makefile

```diff
@@ -0,0 +1,136 @@
+
+# wolfCrypt Additional
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/asm.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/asn.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/blake2s.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/chacha.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/chacha20_poly1305.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/coding.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/compress.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/cpuid.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/cryptocb.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/curve25519.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/curve448.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/ecc_fp.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/eccsi.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/ed25519.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/ed448.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/error.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/fe_448.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/fe_low_mem.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/fe_operations.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/ge_448.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/ge_low_mem.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/ge_operations.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/hash.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/kdf.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/integer.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/logging.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/md5.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/memory.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/pkcs12.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/pkcs7.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/poly1305.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/pwdbased.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/rc2.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/sakke.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/signature.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/srp.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/sp_arm32.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/sp_arm64.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/sp_armthumb.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/sp_c32.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/sp_c64.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/sp_cortexm.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/sp_dsp32.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/sp_int.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/sp_x86_64.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/tfm.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/wc_dsp.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/wc_encrypt.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/wc_pkcs11.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/wc_port.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/wolfevent.c
+CSRCS += $(WOLFSSL_UNPACKNAME)/wolfcrypt/src/wolfmath.c
+
+
+CFLAGS += -DWOLFSSL_USER_SETTINGS
+
+
```

Comment:
remove too

Reply:
that has to be there, wolfssl needs that flag so it knows to read the user_settings.h file instead of the normal configure command

Reply:
or did you mean the white space, that I removed